webknossos/Changelog.md

```diff
@@ -41,6 +41,7 @@ For upgrade instructions, please check the respective _Breaking Changes_ section
 ### Added
 - `Group.add_tree` now also accepts a tree object as a first parameter (instead of only a string). [#891](https://github.com/scalableminds/webknossos-libs/pull/891)
 - `Group.remove_tree_by_id` was added. [#891](https://github.com/scalableminds/webknossos-libs/pull/891)
+- Integrated the WEBKNOSSOS cuber tool into WEBKNOSSOS python library
```

Comment:
Please add a bit more here:
- A list of all old commands and corresponding new commands (e.g. `wkcuber.convert_raw -> webknossos convert-raw`
- Any commands that are gone?
- Other high-level changes the users need to be aware of

Reply:
I added a list of changed, added and removed commands.